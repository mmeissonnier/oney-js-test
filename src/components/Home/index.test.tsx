import React from 'react';
import { render } from '@testing-library/react';
import { Home } from '.';

describe('components/Home', () => {
  it('should render correctly', () => {
    const { container } = render(
      <Home userInfo={{ id: 0, email: 'foo@test.com', name: 'foobar' }} />
    );
    expect(container).toMatchSnapshot();
  });
});
