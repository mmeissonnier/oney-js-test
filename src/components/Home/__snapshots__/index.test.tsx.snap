// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Home should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin: 0;
  color: white;
}

<div>
  <div
    class="c0"
  >
    <h1>
      Home page
    </h1>
    <h2>
      User Infos
    </h2>
    <p>
      {"id":0,"email":"foo@test.com","name":"foobar"}
    </p>
  </div>
</div>
`;
